src/Microsoft.AspNetCore.Routing/DependencyInjection/DispatcherServiceCollectionExtensions.cs

```diff
@@ -2,73 +2,36 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
-using System.Reflection;
 using Microsoft.AspNetCore.Routing;
-using Microsoft.AspNetCore.Routing.EndpointConstraints;
-using Microsoft.AspNetCore.Routing.EndpointFinders;
-using Microsoft.AspNetCore.Routing.Internal;
-using Microsoft.AspNetCore.Routing.Matchers;
-using Microsoft.Extensions.DependencyInjection.Extensions;
-using Microsoft.Extensions.Options;
 
 namespace Microsoft.Extensions.DependencyInjection
 {
+    // This whole class is temporary until we can remove its usage from MVC
     public static class DispatcherServiceCollectionExtensions
     {
+        // This whole class is temporary until we can remove its usage from MVC
```

Comment:
xD

My plan is to react to verify that MVC still passes with these changes, and then:
1. Merge this PR
2. Update branding in MVC
3. Remove these stub `Dispatcher` things from here